netlib/http2/protocol.py

```diff
@@ -93,30 +95,37 @@ def next_stream_id(self):
             self.current_stream_id += 2
         return self.current_stream_id
 
-    def send_frame(self, frame):
-        raw_bytes = frame.to_bytes()
+    def send_frame(self, frm, hide=False):
+        raw_bytes = frm.to_bytes()
         self.tcp_handler.wfile.write(raw_bytes)
         self.tcp_handler.wfile.flush()
+        if not hide and self.dump_frames:  # pragma no cover
+            print(frm.human_readable(">>"))
 
-    def read_frame(self):
+    def read_frame(self, hide=False):
```

Comment:
Pathod/Pathoc has a new CLI argument `--http2-framedump` which will output all frames (incoming & outgoing).
But usually I'm not interested in the connection preface frames - as they are always the same.
Therefore I want to hide only those frame with this argument set to `True`.